frontend/src/pages/AdminPanel.js

```diff
@@ -0,0 +1,1253 @@
+import React, { useState, useEffect } from 'react';
+import axiosClient from '../axiosClient.js';
+import Header from '../components/Header.js';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPen, faTrash, faEye } from '@fortawesome/free-solid-svg-icons';
+
+export default function AdminPanel() {
+  const [activeTab, setActiveTab] = useState(null);
+  const [selectedUser, setSelectedUser] = useState(null);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isDeleteConfirmOpen, setIsDeleteConfirmOpen] = useState(false);
+  
+  {/* Users API */}
+  const [users, setUsers] = useState([]);
```

Comment:
On backend we got a method to add new users, but there's only an edit option on front. We should be able to add users

Reply:
I had a task to only be able to edit users